packages/common/src/errors.js

```diff
@@ -1,6 +1,41 @@
-import { createError } from 'apollo-errors';
+// @flow
 
-const AccountsError = createError('AccountsError', {});
+import type { PasswordLoginUserType } from './types';
 
 // eslint-disable-next-line import/prefer-default-export
-export { AccountsError };
+export class AccountsError extends Error {
+  loginInfo: PasswordLoginUserType;
+  errorCode: string | number;
+  epochTime: number;
+
+  constructor(message: string, loginInfo?: PasswordLoginUserType, errorCode?: string | number) {
```

Comment:
The second parameter, `loginInfo` should be more generally named and typed, other data is passed into here too.